channel-email/src/test/java/com/synopsys/integration/alert/channel/email/attachment/compatibility/ContentKeyTest.java

```diff
@@ -0,0 +1,56 @@
+package com.synopsys.integration.alert.channel.email.attachment.compatibility;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.alert.common.enumeration.ItemOperation;
+
+public class ContentKeyTest {
```

Comment:
Drop the public access modifier from the class  as well as methods as they are not needed for unit tests, we will later get complaints in sonar for code smells otherwise.

Reply:
I ran a full comprehensive build against this branch and saw no increase in code smell count.

Reply:
Fixed